src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Services/MicrosoftGraphServiceExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Graph;
+using Microsoft.Identity.Web;
+
+namespace BlazorServerWeb_CSharp
+{
+    public static class MicrosoftGraphServiceExtensions
+    {
+        /// <summary>
+        /// Adds the Microsoft Graph client as a singleton.
+        /// </summary>
+        /// <param name="services">Service collection.</param>
+        /// <param name="initialScopes">Initial scopes.</param>
+        /// <param name="graphBaseUrl">Base URL for Microsoft graph. This can be
+        /// changed for instance for applications running in national clouds</param>
+        public static IServiceCollection AddMicrosoftGraph(this IServiceCollection services,
+                                             IEnumerable<string> initialScopes,
+                                             string graphBaseUrl = "https://graph.microsoft.com/v1.0")
+        {
```

Comment:
nit: We don't indent code like this in our templates. We prefer avoiding indentation all together and if necessary we just indent one tab/4spaces to the right

Reply:
I'll leave the nits for later, There are lots of style fixes we plan to do as part of API cleanup in the future.